dev/src/dev/toucan2_monitor.clj

```diff
@@ -42,6 +54,12 @@
 (defn start!
   "Start tracking queries."
   []
+  (tu.log/set-ns-log-level! *ns* :debug)
+  (reset! log-future (future
```

Comment:
What if there is already a future running? I think this should we a swap that also cancels it if it's not nil.

Reply:
updated so that this handle re-entry